app/Resources/views/editCounter/general_stats.html.twig

```diff
@@ -126,7 +126,12 @@
                 <td>{{ ec.averageRevisionsPerPage }}</td>
             </tr>
             <tr>
-                <td>{{ msg('pages-created') }}</td>
+                <td>
+                    <a class="glyphicon glyphicon-info-sign" target="_blank" title="{{ msg('pages-created-tooltip') }}"
+                       href="https://xtools.readthedocs.io/en/latest/tools/editcounter.html#general-statistics">
+                    </a>
+                    {{ msg('pages-created') }}
+                </td>
```

Comment:
This works but the only way to see the translated message is to hover and _wait_ over the info icon. Most will click on the link which is only in English. If you want we could use our own tooltip system which will make the translatable message more prominent https://github.com/x-tools/xtools/blob/master/app/Resources/views/macros/forms.html.twig#L5-L11